app/services/main.py

```diff
@@ -21,8 +21,10 @@
 #  SOFTWARE.
 
 from fastapi import FastAPI
+from fastapi.staticfiles import StaticFiles
 
 from .an import an
+from .mobilize import mobilize
```

Comment:
Here's a tiny, insignificant issue for you :)... please invert lines 27 and 28 so imports of the same "dot depth" are alphabetized.  (I wouldn't ask you to do this if there weren't more substantive changes that also have to happen, really I promise!)